server/etes/admin.py

```diff
@@ -1,3 +1,7 @@
 from django.contrib import admin
+from .models import User, Ticket, Event
 
 # Register your models here.
+admin.site.register(User)
+admin.site.register(Ticket)
+admin.site.register(Event)
```

Comment:
Fix new line

Reply:
I recommend adding `"files.insertFinalNewline": true` to `.vscode/settings.json` so VSCode will handle this in the future.